relaymessage.go

```diff
@@ -0,0 +1,117 @@
+package dhcp6
+
+// Packet represents a raw DHCPv6 relay message, using the format described in RFC 3315,
+// Section 7.
+// Relay Agent/Server Message Formats
```

Comment:
Remove this section copied from the RFC.  It probably won't be displayed very well in GoDoc and the RFC can be referenced for more details if needed.